src/modules/fullpageos/filesystem/home/pi/scripts/start_chromium_browser

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
 
+SERIAL=`cat /proc/cpuinfo | perl -n -e '/^Serial[ ]*: ([0-9a-f]{16})$/ && print "$1\n"'`
+
 # Standard behavior - runs chrome
-chromium-browser --kiosk --touch-events=enabled --disable-pinch --noerrdialogs --disable-session-crashed-bubble --app=$(head -n 1 /boot/fullpageos.txt)
+chromium-browser --kiosk --touch-events=enabled --disable-pinch --noerrdialogs --disable-session-crashed-bubble --app=$(head -n 1 /boot/fullpageos.txt | sed -e "s/{serial}/${MYVAR}/g")
```

Comment:
shouldn't you replace with ${SERIAL} instead of ${MYVAR} ?

Reply:
Oops! That was a wrong copy-past from my test. Fixed it now